<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rFSM Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainsplitterH">
      <property name="font">
       <font>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsViewEc" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>20</horstretch>
         <verstretch>15</verstretch>
        </sizepolicy>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBoxEvent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Events</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="pushButtonSendEvent">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBoxEvents"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxEventQueue">
          <property name="title">
           <string>Event Queue</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="nodesTreeWidgetEvent">
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Time</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Event</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxLog">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="nodesTreeWidgetLog">
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Time</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Message</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_LoadrFSM"/>
    <addaction name="action_Save_project"/>
    <addaction name="separator"/>
    <addaction name="actionExport_scene"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentaion"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <addaction name="actionOrthogonal"/>
     <addaction name="actionPolyline"/>
     <addaction name="actionCurved"/>
     <addaction name="actionLine"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuLayout"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionStep"/>
    <addaction name="actionRun"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_LoadrFSM"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStep"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_LoadrFSM">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/file-new.svg</normaloff>:/icons/resources/file-new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Load rFSM...</string>
   </property>
   <property name="toolTip">
    <string>Load rFSM</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionDocumentaion">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/help-info.svg</normaloff>:/icons/resources/help-info.svg</iconset>
   </property>
   <property name="text">
    <string>Documentaion...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_Save_project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/file-save.svg</normaloff>:/icons/resources/file-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save project</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orthogonal</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionCurved">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Curved</string>
   </property>
  </action>
  <action name="actionSubgraph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Subgraph</string>
   </property>
  </action>
  <action name="actionExport_scene">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/module.svg</normaloff>:/icons/resources/module.svg</iconset>
   </property>
   <property name="text">
    <string>Export scene...</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/actions-media-seek-forward.svg</normaloff>:/icons/resources/actions-media-seek-forward.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Step</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/actions-media-play.svg</normaloff>:/icons/resources/actions-media-play.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="ress.qrc">
     <normaloff>:/icons/resources/actions-refresh.svg</normaloff>:/icons/resources/actions-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>Rese&amp;t</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewEc</class>
   <extends>QGraphicsView</extends>
   <header>QGraphicsViewEc.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ress.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
